cmd/core-command/res/configuration.toml

```diff
@@ -1,34 +1,26 @@
-ConsulProfilesActive = 'go'
-ReadMaxLimit = 100
-ConsulPort = 8500
-ServiceTimeout = 5000
+[Service]
+BootTimeout = 30000
+ClientMonitor = 15000
 CheckInterval = '10s'
-ServiceAddress = 'localhost'
-ServicePort = 48082
-DeviceServiceProtocol = 'http'
-AppOpenMsg = 'This is the Core Command Micro Service'
-URLProtocol = 'http://'
-URLDevicePath = '/api/v1/device'
-ConsulHost = 'localhost'
-ConsulCheckAddress = 'http://localhost:48082/api/v1/ping'
-EnableRemoteLogging = false
-LogFile = './logs/edgex-core-command.log'
-LoggingRemoteURL = 'http://localhost:48061/api/v1/logs'
-MetaAddressableURL = 'http://localhost:48081/api/v1/addressable'
-MetaAddressablePath = '/api/v1/addressable'
-MetaDeviceServiceURL = 'http://localhost:48081/api/v1/deviceservice'
-MetaDeviceServicePath = '/api/v1/deviceservice'
-MetaDeviceProfileURL = 'http://localhost:48081/api/v1/deviceprofile'
-MetaDeviceProfilePath = '/api/v1/deviceprofile'
-MetaDeviceURL = 'http://localhost:48081/api/v1/device'
-MetaDevicePath = '/api/v1/device'
-MetaDeviceReportURL = 'http://localhost:48081/api/v1/devicereport'
-MetaDeviceReportPath = '/api/v1/devicereport'
-MetaCommandURL = 'http://localhost:48081/api/v1/command'
-MetaCommandPath = '/api/v1/command'
-MetaEventURL = 'http://localhost:48081/api/v1/event'
-MetaEventPath = '/api/v1/event'
-MetaScheduleURL = 'http://localhost:48081/api/v1/schedule'
-MetaSchedulePath = '/api/v1/schedule'
-MetaProvisionWatcherURL = 'http://localhost:48081/api/v1/provisionwatcher'
-MetaProvisionWatcherPath = '/api/v1/provisionwatcher'
+Host = 'localhost' #Is this used?
+Port = 48082
+Protocol = 'http' #Is this used?
+ReadMaxLimit = 100
+StartupMsg = 'This is the Core Command Micro Service'
+Timeout = 5000
+
+[Registry]
+Host = 'localhost'
+Port = 8500
+Type = 'consul'
+
+[Logging]
+EnableRemote = false
+File = './logs/edgex-core-command.log'
+RemoteURL = 'http://localhost:48061/api/v1/logs' #Should this also be decomposed -- protocol / host / port?
```

Comment:
I think should be decomposed and /api/v1/logs moved into pkg/clients/constants.go

Reply:
This is included in the updated PR